src/Ledger/Utxow.lagda

```diff
@@ -161,9 +161,10 @@ data _⊢_⇀⦇_,UTXOW⦈_ where
         neededHashes      = scriptsNeeded utxo txb
         txdatsHashes      = dom txdats
         allOutHashes      = getDataHashes (range txouts)
+        nonRefScripts     = mapPartial isInj₁ (txscripts tx utxo)
```

Comment:
I think it should be called something like `nativeScripts`. `nonRefScripts` doesn't make much sense here because `txscripts` gives both the reference scripts and scripts in the `txins`.